# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libcap/libcap-2.06-parallel-make-hotfix.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Submitted by: Christian Wiese <morfoh@opensde.org>
Date: 2008-02-23
Initial Package Version: 2.06
Upstream Status: submitted
Origin: OpenSDE
Description: Fixes build issues while using more than one make job, assuring
             that cap_names.h is generated before compiling cap_text.c. 

--- libcap-2.06/libcap/Makefile.orig	2008-02-23 03:00:20.000000000 +0000
+++ libcap-2.06/libcap/Makefile	2008-02-23 02:59:34.000000000 +0000
@@ -51,7 +51,7 @@
 %.o: %.c $(INCLS)
 	$(CC) $(CFLAGS) -c $< -o $@
 
-cap_text.o: cap_text.c $(USE_GPERF_OUTPUT)
+cap_text.o: cap_text.c $(USE_GPERF_OUTPUT) $(INCLS)
 	$(CC) $(CFLAGS) $(INCLUDE_GPERF_OUTPUT) -c $< -o $@
 
 install: all
